news/2023/2023-01-11-new-featured-artist-euchaeta.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: "New Featured Artist: Euchaeta"
+date: 2023-01-11 11:00:00 +0000
+---
+
+**Euchaeta** brings some novelty into our Featured Artist catalogue!
+
+![](https://assets.ppy.sh/artists/333/header.jpg)
+
+Japanese-style instrumentation, folk-inspired vocals, and VOCALOID are some main ingredients behind **Euchaeta**'s eclectic music. A master of his craft, **Euchaeta** has been designing musical journeys since 2010, and they've only recently been noticed by our circle clicking community!
```

Comment:
```suggestion
Japanese-style instrumentation, folk-inspired vocals, and VOCALOID are some main ingredients behind **Euchaeta**'s eclectic music. A master of his craft, **Euchaeta** has been designing musical journeys since 2010, and they've only recently been noticed by our circle-clicking community!
```